charts/connect/templates/connect-serviceaccount.yaml

```
@@ -0,0 +1,13 @@
+{{- if eq (tpl (.Values.operator.serviceAccount.create | toString) .) "true" -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.operator.serviceAccount.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "onepassword-connect.labels" . | nindent 4 }}
+  {{- with .Values.operator.serviceAccount.annotations }}
```

Seems it should it be
```suggestion
  {{- with .Values.connect.serviceAccount.annotations }}
```